src/lib/CLValue2/Tuple.ts

```diff
@@ -1,9 +1,18 @@
+import { Ok, Err } from "ts-results";
```

Comment:
> // TBD: Maybe replace this with classic implementation, it will be easier to validate by compiler eg. new Tuple1<Bool>(), new Tuple2<Bool, I32>();

Yes. I like how Numbers are implemented.